src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.csproj

```diff
@@ -6,17 +6,25 @@
          to its own obsolete API. -->
     <NoWarn>618</NoWarn>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <AssemblyVersion>4.0.21.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
```

Comment:
Due to the number of internal TFS build breaks any version bumps like this need to be made on the TFS side. So please continue the review here but apply them your changes to TFS and check-in the changes from there.